build(aio): update to @angular/cli@1.0.0-beta.31

IgorMinar opened the pull request.

IgorMinar committed:
build(aio): update to @angular/cli@1.0.0-beta.31

IgorMinar committed:
build(aio): update to @angular/core@2.4.7

upgrade command used:

```
yarn upgrade @angular/{core,common,compiler,forms,http,platform-browser,platform-browser-dynamic,router,compiler-cli}
```

wardbell reviewed (approved):
LGTM.  I ran my version of the DocViewer (e2e, ng test, ng serve) and all was well.

IgorMinar committed:
ci: exclude aio/ from build-and-ci pullapprove group

the aio/ has a separate build and ci setup that doesn't affect main repo, so it doesn't make
sense for the build-and-ci group to approve changes to that separate setup.
